contracts/BaseLaunchpeg.sol

```diff
@@ -462,14 +370,18 @@ abstract contract BaseLaunchpeg is
         emit DevMint(msg.sender, _quantity);
     }
 
-    /// @notice Withdraw AVAX to the contract owner
+    /// @notice Withdraw AVAX to the given recipient
     /// @param _to Recipient of the earned AVAX
     function withdrawAVAX(address _to)
         external
         override
-        onlyOwner
+        onlyOwnerOrRole(PROJECT_OWNER_ROLE)
         nonReentrant
     {
+        if (withdrawAVAXStartTime > block.timestamp) {
+            revert Launchpeg__WithdrawAVAXNotAvailable();
+        }
```

Comment:
It should revert on `withdrawAVAXStartTime == 0`, as it would mean it wasn't initialized yet